pkg/controller/rest/verifiable/operation.go

```diff
@@ -155,23 +159,14 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
 	rest.Execute(o.command.GeneratePresentation, rw, req.Body)
 }
 
-// GeneratePresentationByID swagger:route GET /verifiable/credential/{id}/presentation verifiable presentationByIDReq
+// GeneratePresentationByID swagger:route GET /verifiable/credential/presentationByID
```

Comment:
why is this having a suffix of ByID?

Reply:
```
	generatePresentationPath     = /verifiable/presentation/generate
	generatePresentationByIDPath = /verifiable/credential/{id}/presentation
```

This is what we have in the master. Now getPresentationByID no longer have. query parameters.  I think i should change this to - :  It is post request which take VCID and DID ID as request body

`generatePresentationByIDPath = /verifiable/credential/id/presentation`